src/components/Dropdown/components/menu/menu.jsx

```diff
@@ -7,8 +7,11 @@ const Menu = ({ children, Renderer, selectProps, dropdownMenuWrapperClassName, .
   const rendererProps = { children, selectProps, ...props };
   const withFixedPosition =
     selectProps?.selectProps?.insideOverflowContainer || selectProps?.selectProps?.insideOverflowWithTransformContainer;
-  // Temporary fix for menu animation is above the select when using menuPortalTarget
-  const withoutAnimation = !!selectProps?.menuPortalTarget;
+  // Temporary fix, which disables animation :
+  // - when using menuPortalTarget or popupsContainerSelector there are issues that animation goes above the select preventing from tapping it (pr #1543)
+  // - when using menuPosition="fixed" (e.g. inside of Modal) not whole list is visible for scroll cause of it (pr #1738)
+  const withoutAnimation =
+    !!selectProps?.menuPortalTarget || !!selectProps.popupsContainerSelector || selectProps?.menuPosition === "fixed";
```

Comment:
`menuPortalTarget` and `popupsContainerSelector` is basically the same, so it should have been enabled for them before
For the `menuPosition=fixed` this animation was ruining the correct scrolling from the bottom to the top of all the options